rapids-cmake/cpm/libcudacxx.cmake

```diff
@@ -103,14 +103,18 @@ function(rapids_cpm_libcudacxx)
     # the headers to `include/rapids/libcudacxx`
     include(GNUInstallDirs)
     set(CMAKE_INSTALL_INCLUDEDIR "${CMAKE_INSTALL_INCLUDEDIR}/rapids/libcudacxx")
+    set(CMAKE_INSTALL_LIBDIR "${CMAKE_INSTALL_LIBDIR}/rapids/")
+
+    include("${rapids-cmake-dir}/export/find_package_root.cmake")
+    rapids_export_find_package_root(INSTALL libcudacxx
+                                    [=[${CMAKE_CURRENT_LIST_DIR}/../../rapids/cmake/libcudacxx]=]
+                                    ${_RAPIDS_INSTALL_EXPORT_SET})
 
     # libcudacxx 1.8 has a bug where it doesn't generate proper exclude rules for the
```

Comment:
Can we remove this whole block when we bump to the latest libcudacxx? I want to know since I'll be updating CCCL soon.

Reply:
I have removed all of the patching here. We will still need to patch libcudacxx ( even 2.0+ ) but the match set will be fairly minimal. The libcudacxx header search wasn't set up like thrust/cub and doesn't work with custom CMAKE_INSTALL_INCLUDEDIR values.

Reply:
Okay, thanks! I approved the PR.